artifactory/utils/argsutils_test.go

```diff
@@ -263,4 +263,12 @@ func TestParseArgs(t *testing.T) {
 	want = []string{`--build-name=jfrog jfrog`, "--build-number=1"}
```

Comment:
The above *want* is good for the following two tests as well. No need to declare it again.

Reply:
Removed